packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/generatedAnswer.html

```diff
@@ -0,0 +1,39 @@
+<template>
+  <template lwc:if={shouldDisplayGeneratedAnswer}>
+    <div
+      data-cy="generated-answer__card"
+      class="slds-box generated-answer__card"
+    >
+      <div
+        class="slds-var-m-bottom_medium slds-grid generated-answer__card-header"
+      >
+        <span class="generated-answer__badge slds-var-p-around_x-small"
+          >{labels.generatedAnswerForYou}</span
+        >
+        <c-quantic-feedback
+          state={feedbackState}
+          onlike={handleLike}
+          ondislike={handleDislike}
+          like-icon-name="utility:like"
+          like-label=""
+          dislike-icon-name="utility:dislike"
+          dislike-label=""
+          size="x-small"
+          question=""
+          hide-explain-why-button
+          data-cy="generated-answer__feedback"
+        ></c-quantic-feedback>
+      </div>
+      <div data-cy="generated-answer__content" class={generatedContentClass}>
+        {answer}
+      </div>
```

Comment:
Are the references part of the answer? Maybe we should extract them in a seperate div since they differ from the mockup:

here:
![image](https://github.com/coveo/ui-kit/assets/73316533/072c0283-c5d7-4379-a79c-9889aeca5a82)



mockup:
<img width="333" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/85f590e5-91de-40bb-b7fa-62ef099ae577">

Reply:
For the first remark, I like the solution you came up with! 👍 

For the second remark, I meant setting a max-width to the generated answer so it stops aligned with the feedback like in the mockup and not justified (bad choice of word on my end). I do agree with you that it looks weird with the justification x)

Reply:
Ah I think it was just a coincidence that the text in the mockups stopped with the like button, cause the text should have the same amount of padding in both sides left and right.